Assets/General/Audio/FMODScripts/IntervalAmbienceEvents.cs

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+// File Name:       IntervalAmbienceEvents.cs
+// Author:          Mark Hanson
+// Creation Date:   11/14/2024
+//
+// Description:     Handler of random ambient sound effects at random times
+******************************************************************************/
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using FMODUnity;
+using Random = UnityEngine.Random;
```

Comment:
I didn't know you could specify the random like this. This is neat. No changes, just good to know :D